tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3784,20 +3815,52 @@ <h2>Show similarity to selected datapoint</h2>
           }
         },
 
+        isSameInferenceClass_: function(val1, val2) {
+          return this.isRegression_(this.modelType)
+            ? Math.abs(val1 - val2) < this.minCounterfactualValueDist
+            : val1 == val2;
+        },
+
+        adjustMaxCounterfactualValueDist_: function(selected, valueStr) {
+          this.maxCounterfactualValueDist = Math.max(
+            this.distanceStats_[valueStr].max -
+              this.visdata[selected][valueStr],
```

Comment:
Do not know about the nature of `distanceStats` and `visData`: can all of these values be negative? Can `visData` be lower than both max and min of the `distanceStats`? Can this be doing Math.max on, for example, `Math.max(-2, -3)`?

Reply:
`distanceStats_` is based on the `visData` data, so that:
`distanceStats_[valueStr].min` <= `visdata[selected][valueStr]` <= `distanceStats_[valueStr].max`

This `Math.max` op then receives only non-negative values.